### DISCUSSION TITLE
Query about licensing

### DISCUSSION THREAD

**Discussion by SyazwanSamad** [OpenRadioss/ModelExchange#9] [Q&A] [answered]
Hi, I have made the following model to showcase about damage initialization based on strain histories. The vehicle crash model is made exclusively with the purpose to present in ATC ASEAN.
![DYNAMIC](https://github.com/OpenRadioss/ModelExchange/assets/113438036/f3aef89e-e23c-4239-b374-b1178e0a1826)

However, due to the model is derived base on the publicly available finite element model developed by CCSA, I'm not sure the status of the license. 

In the comments, they only requested CCSA and FHWA to be acknowledged for any use of the FE model. Can this model still be uploaded?

**Comment by PaulAltair** [comment]
Hi @SyazwanSamad , yes, this should be fine, I suggest you include in the model header, acknowledgment of the model derivation as suggested by CCSA/FHWA, if you look in the headers of the HPC benchmark models, you will see exactly this type of acknowledgment since they are also derived from publicly available source.

**Reply by SyazwanSamad** [reply]
Hi @PaulAltair 
Thank you for the prompt reply. I understand, I can just follow the example from the HPC benchmark models.
However, one more thing. What should I put for the Copyright Holder?
  ---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|
   Insert Model Name ("Model")
   Copyright (C) Insert Copyright Year **Insert Copyright Holder** ("Holder")
   Model is licensed by Holder under CC BY-NC 4.0
   (https://creativecommons.org/licenses/by-nc/4.0/legalcode).  
  ---1----|----2----|----3----|----4----|----5----|----6----|----7----|----8----|----9----|---10----|

**Reply by PaulAltair** [reply, marked as answer]
You can put yourself (just your github username or as much additional information as you want) or your institution/organisation if you prefer (or are required to)
